docs/guide/examplegen.md

```diff
@@ -111,6 +111,47 @@ Please refer to
 [proto/example_gen.proto](https://github.com/tensorflow/tfx/blob/master/tfx/proto/example_gen.proto)
 for details.
 
+### Splitting Method
+
+When using `hash_buckets` splitting method, instead of the entire record,
+one can use a feature for partitioning the examples. If it's not empty,
```

Comment:
`if it's not empty` is a bit confusing, e.g., if a string feature value is "", this is a valid case. 

consider rewording it to If the feature is present,

Reply:
got it, thanks for the catch